mongodbatlas/table_mongodbatlas_team.go

```diff
@@ -0,0 +1,104 @@
+package mongodbatlas
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func tableAtlasTeam(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "table_mongodbatlas_team",
+		Description: "",
+		List: &plugin.ListConfig{
+			Hydrate:       listTeams,
+			ParentHydrate: listAtlasProjects,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Description: "Unique identifier for the team.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "name",
+				Description: "Name of the team.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: "Title of the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+		},
+	}
+}
+
+func listTeams(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func listMongoDBAtlasTeams(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Function names should be consistent, and we usually include the provider/service name in the functions as well